charts/seed-operatingsystemconfig/original/templates/osc.yaml

```diff
@@ -19,11 +23,22 @@ spec:
 {{ include "update-ca-certs" . | indent 2 }}
 {{ include "docker-logrotate" . | indent 2 }}
```

Comment:
What about these two units (docker-logrotate)? Are they still valid even when using containerd?

Reply:
Valid, because docker is still running. but maybe not that useful anymore

Reply:
Do we need a similar mechanism for containerd? How does the log rotation work here? cc @ialidzhikov

Reply:
When moving to CRI, The Kubelet is responsible for rotating the containers logs.

Reply:
logrotation of containerd was added instead of kubelet rotation.